tools/provision.ps1

```diff
@@ -169,7 +205,10 @@ function Install-PipPackage {
     Exit -1
   }
   Write-Host " => Attempting to install Python packages" -foregroundcolor DarkYellow
-  $env:Path = "$env:Path;$env:HOMEDRIVE\tools\python2;$env:HOMEDRIVE\tools\python2\Scripts"
+  $pythonPath = "$env:HOMEDRIVE\\tools\\python2"
+  $pythonScriptsPath = "$env:HOMEDRIVE\\tools\\python2\\Scripts"
```

Comment:
This isn't working. To match against strings that already exist in the path I need the `\\`, however when adding that character isn't treated as the escape character. Got something better coming here ina bit.